### ISSUE TITLE
Feature request: Context menu entry for preview of edited files in new browser tab

### ISSUE THREAD

**Issue by springerpk** [Atheos/Atheos#199]
**Is your feature request related to a problem? Please describe.**
If I want to preview an edited file, I have to open a new browser tab and type in the full address of the file.

**Describe the solution you'd like**
Add an entry in the context menu of a file which opens the file in a new browser tab.

**Additional context**
Telaaedifex team's custom version of Codiad did it here:
https://github.com/xevidos/codiad/blob/master/components/filemanager/context_menu.json

**Comment by HLSiira** [comment]
It actually used to be a feature originally but I never found it particularly effective and functional due to the method that Atheos utilized Absolute vs Relative paths. I can reimplement it but it'll take some time as the pathing is little bit more complex now.

**Comment by HLSiira** [comment]
I don't know what you mean by this last comment, can you elaborate?

I never really used this in Codiad so I don't have much context on it

**Comment by 3642066** [comment]
█▀▄ █▀▀▄ █▀▀ ▐▌─▐▌ ▀ █▀▀ █───█ 
█─█ █▐█▀ █▀▀ ─▀▄▀─ █ █▀▀ █─█─█ 
█▀─ ▀─▀▀ ▀▀▀ ──▀── ▀ ▀▀▀ ─▀─▀─ 
Please help about this feature, its usefull .
Also the extraction plugin, please .
I still using codiad because of this .

**Comment by HLSiira** [comment]
I didn't realize that that repo had any updates made to it. I'll try to add the feature soon but it's going to take a while to figure out the source code from that repo. They modified a TON of things and it's hard to follow why.

**Comment by svenn71** [comment]
I use Codiad for asciidoc and markdown files, and the file preview together with the asciidoc and markdown preview plugins for the browser is a really conveniant documentation flow.

**Comment by inglesuniversal** [comment]
Hope this improvement is added soon! 

Even an internal <a href="current_path/workspace/project/opened_file.php" target="_blank"> could help </a>

**Comment by inglesuniversal** [comment]
Great news... Do I need to update the entire package for this to work or only a particular file or folder?

Please comment on this.

Thanks

**Comment by HLSiira** [comment]
It's best to update to the latest push to `main`, as there have been a lot of changes to Atheos and I can't promise a single file change will work as expected, just due to the changes to the APIs of each component over the months.
